packages/react-native/src/Authenticator/Defaults/VerifyUser/VerifyUser.tsx

```diff
@@ -15,6 +19,19 @@ const COMPONENT_NAME = 'VerifyUser';
 const { getSkipText, getVerifyText, getAccountRecoveryInfoText } =
   authenticatorTextUtil;
 
+const censorContactInformation = (name: string, value: string): string => {
+  let censoredVal = value;
+  if (name === 'email') {
+    const splitEmail = value.split('@');
+    const censoredName = censorAllButFirstAndLast(splitEmail[0]);
+
+    censoredVal = `${censoredName}@${splitEmail[1]}`;
+  } else if (name === 'phone') {
+    censoredVal = censorPhoneNumber(value);
+  }
+  return censoredVal;
```

Comment:
Will this fallback `return` ever be called?

Reply:
Fallback return? how so? it's the only return.

Reply:
I'm sorry, I misread the code 😓  Please disregard!